packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -61,15 +61,26 @@
       "single"
     ],
     "trailing-comma": false,
-    "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "banana-in-box": true,
+    "component-class-suffix": true,
+    "contextual-life-cycle": true,
+    "directive-class-suffix": true,
+    "enforce-component-selector": true,
+    "no-attribute-parameter-decorator": true,
+    "no-conflicting-life-cycle-hooks": true,
     "no-input-rename": true,
+    "no-output-named-after-standard-event": true,
+    "no-output-on-prefix": true,
     "no-output-rename": true,
+    "no-queries-parameter": true,
```

Comment:
We can drop this one.

Reply:
Out of curiosity, why not keep this one? Using `queries` in the component decorator feels a bit weird no?